code/obj/item/clothing/suits.dm

```diff
@@ -1283,32 +1283,55 @@
 	item_state = "space-neon"
 
 /obj/item/clothing/suit/space/custom // Used for nanofabs
+	icon_state = "spacemat"
+	inhand_image_icon = "s_suit"
+	item_state = "spacemat"
 	name = "bespoke space suit"
-	desc = "A suit that protects against low pressure environments, custom-made just for you!"
-	onMaterialChanged()
-		if(src.material)
-			if(material.hasProperty("thermal"))
-				var/prot = 100 - material.getProperty("thermal")
+	desc = "A custom built suit that protects your soft tissues from hard vacuum."
+	proc/setupSuitProp(var/datum/material/F, var/datum/material/R) // f is fabric, decides protection, R is reinforcment, decides melee prot
+
+		if(F)
```

Comment:
This checks for `F` not being null but the code that works with `R` is also there. That doesn't make much sense.